hbp_knowledge/receptors/king2015.bel

```diff
@@ -94,250 +94,283 @@ p(HGNC:GNAI1) -> complex(p(HGNC:GNAI1), p(HGNC:CHRNA7))
 p(HGNC:GNAI2) -> complex(p(HGNC:GNAI2), p(HGNC:CHRNA7))
 
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "An association between α7 nAChRs and Gβ was observed in the
 prefrontal cortex and hippocampus, whereas little to no detection of Gαo was
 seen in the adult brain (Fig. 1A)."
-
+SET Confidence = "High"
 SET MeSHAnatomy = {"Prefrontal Cortex", "Hippocampus"}
 
 p(HGNCGENEFAMILY:"G protein subunits beta") -> complex(p(FPLX:"G_protein"), p(HGNC:CHRNA7))
 
 UNSET MeSHAnatomy
+UNSET Confidence
 
 SET Evidence = "Gel lanes of the BgtxC were divided into 3 molecular mass
 fractions: F1, 190-90 kDa; F2, 89-45 kDa; and F3, 44-15 kDa (Fig. 1C). Proteomic
 analysis was performed on the individual fractions using tryptic in-gel
 digestion followed by high-performance LC-ESI. A cohort of G proteins was
 identified in the F2 and F3 fractions (Fig. 1D). These G proteins were not
 detected in BgtxC from α7−/− mice (data not shown)."
-
+SET Confidence = "High"
 composite(p(HGNC:CHRNA7), p(MESH:"Heterotrimeric GTP-Binding Proteins"), p(HBP:"alpha-Bungarotoxin")) -> complex(p(MESH:"Heterotrimeric GTP-Binding Proteins"), a(HBP:"alpha-Bungarotoxin"))
+UNSET Confidence
 
 SET Evidence = "Sequence alignment of mammalian nAChRs revealed a conservation
 of residues at the GPBC site in the M3-M4 loop (Fig. 2A). Based on amino acid
 similarity we found that α7 nAChRs possess 88% homology with the GyR1 at the
 GPBC (Fig. 2A)."
 # Typo here in GyR1 , should be GlyR1 which means Glycine Receptor 1 (GLRA1)
-
+SET Confidence = "High"
 p(HBP:"GPBC, alpha-7 nAChR") orthologous p(HBP:"GPBC, Glycine Receptor 1")
+UNSET Confidence
 
 SET Evidence = "The findings
 confirm that the mutation in α7345–348A does not interfere with the trafficking
 or expression of the nAChR."
-
+SET Confidence = "Medium"
 p(HGNC:CHRNA7) -> bp(HBP:"nAChR assembly")
-p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")) -> bp(HBP:"nAChR assembly")
+p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")) cnc bp(HBP:"nAChR assembly")
+#-> to cnc because mutation in α7345–348A does not interfere
+UNSET Confidence
 
 SET Evidence = "Western blot analysis confirms that transfection of α7 nAChRs
 augments total α7 subunit expression in PC12 cells, which express endogenous α7
 nAChRs. Transfection with α7 (α7+) increased the immunoreactive α7 signal by over
 60% from endogenous mock-transfected control cells, whereas transfection with the
 mutant α7345–348A increased the total α7 signal by 48% over the endogenous α7
 from control cells (Fig. 3A)."
-
+SET Confidence = "Low"
 SET MeSHAnatomy = "PC12 Cells"
 
 p(HGNC:CHRNA7) -> p(HGNC:CHRNA7)
 p(HGNC:CHRNA7, var("p.345A"), var("p.346A"), var("p.347A"), var("p.348A")) -> p(HGNC:CHRNA7)
-
+#percentage difference in different conditions hold importance and with above Statements it is difficult to do justice.
```

Comment:
it is not possible to code quantitative information with bel, so just stick with upregulation